macros/src/main/scala-2/com/softwaremill/macwire/internals/DependencyResolver.scala

```diff
@@ -4,7 +4,7 @@ import com.softwaremill.macwire.internals.EligibleValuesFinder.Scope.LocalForwar
 
 import scala.reflect.macros.blackbox
 
-private[macwire] class DependencyResolver[C <: blackbox.Context](val c: C, debug: Logger) {
+private[macwire] class DependencyResolver[C <: blackbox.Context, TypeC <: C#Type, TreeC <: C#Tree](val c: C, debug: Logger)(resolutionFallback: TypeC => TreeC) {
```

Comment:
I think we should be able to use `c`-dependent types in the second argument list